File: lab1/circle/circle.cpp
#include "circle.h"
#include <cmath>

Circle::Circle(float R)
{
    Radius= R;
}
float Circle::getArea(void)
{
    return M_PI*pow(Radius,2);
}
float Circle::getPerimeter(void)
{
    return 2*M_PI*Radius;
}

File: lab1/circle.java
package cirlce;

public class circle {
	private float radius;
	circle (float R)
	{
		radius= R;
	}
	double perimeter()
	{
		return 2*Math.PI*radius;
	}
	double area()
	{
		return 2*Math.PI*Math.pow(radius, 2);
	}
	public static void main(String args[])
	{
		circle cir1= new circle(12);
		circle cir2= new circle(15);
		System.out.println("perimeter of cir1 is: "+cir1.perimeter());
		System.out.println("area of cir1 is: "+cir1.area());
		System.out.println("perimeter of cir2 is: "+cir2.perimeter());
		System.out.println("area of cir2 is: "+cir2.area());
	}
}

